From f169adab50412f2f3c4a8536652c2376e97e95b1 Mon Sep 17 00:00:00 2001
From: Patrik <development@finworks.biz>
Date: Mon, 17 Nov 2025 09:07:23 +0000
Subject: [PATCH 3/9] Removing Pharo baseline section to debug GemStone part

---
 .../instance/baseline..st                     | 37 -------------------
 1 file changed, 37 deletions(-)

diff --git a/repository/BaselineOfZincHTTPComponents.package/BaselineOfZincHTTPComponents.class/instance/baseline..st b/repository/BaselineOfZincHTTPComponents.package/BaselineOfZincHTTPComponents.class/instance/baseline..st
index 6075cfa1..cd6c747c 100644
--- a/repository/BaselineOfZincHTTPComponents.package/BaselineOfZincHTTPComponents.class/instance/baseline..st
+++ b/repository/BaselineOfZincHTTPComponents.package/BaselineOfZincHTTPComponents.class/instance/baseline..st
@@ -43,43 +43,6 @@ baseline: spec
             group: 'REST' with: #('Zinc-REST');
             group: 'Server-Sent-Events' with: #('Zinc-Server-Sent-Events') ].
 
-    spec for: #'pharo' do: [
-        spec baseline: 'NeoJSON'
-                 with: [ spec
-                             loads: #('Core');
-                             repository: 'github://svenvc/NeoJSON:master/repository' ];
-             baseline: 'XMLParser'
-                 with: [ spec
-                             loads: #('Core');
-                             repository: 'github://pharo-contributions/XML-XMLParser:master/src' ];
-            baseline: 'Zodiac Project'
-                with: [ spec
-                            className: 'BaselineOfZodiac';
-                            loads: #('Core' 'Tests');
-                            repository: 'github://svenvc/zodiac:master/repository' ].
-        spec
-            package: 'Zinc-Pharo-Character-Encoding-Core' with: [ spec requires: #('Zinc-Character-Encoding-Core') ];
-            package: 'Zinc-Pharo-Character-Encoding-Tests' with: [ spec requires: #('Zinc-Character-Encoding-Tests') ];
-            package: 'Zinc-Pharo-Resource-Meta-Core' with: [ spec requires: #('Zinc-Resource-Meta-Core' 'Zinc-Pharo-Character-Encoding-Core') ];
-            package: 'Zinc-Pharo-HTTP' with: [ spec postLoadDoIt: #recompileZnLogEventHierarchy;
-                                                    requires: #('Zinc-Pharo-Character-Encoding-Core' 'Zinc-Pharo-Resource-Meta-Core' 'Zinc-HTTP') ];
-            package: 'Zinc-Pharo-HTTP-Examples' with: [ spec requires: #('Zinc-HTTP-Examples') ];
-            package: 'Zinc-Pharo-Tests' with: [ spec requires: #('Zinc-Tests') ].
-        spec
-            package: 'Zinc-Pharo-WebSocket-Core' with: [ spec requires: #('Zinc-Pharo-HTTP') ];
-            package: 'Zinc-Pharo-WebSocket-Tests' with: [ spec requires: #('Zinc-Pharo-WebSocket-Core') ];
-            package: 'Zinc-Zodiac-Core' with: [ spec requires: #('Zodiac Project') ];
-            package: 'Zinc-Zodiac-Tests' with: [ spec requires: #('Zinc-Zodiac-Core') ];
-            package: 'Zinc-Pharo-Zodiac-Tests' with: [ spec requires: #('Zinc-Zodiac-Tests') ].
-        spec
-            group: 'Core' with: #('Zinc-Pharo-HTTP' 'Zinc-Pharo-HTTP-Examples');
-            group: 'Tests' with: #('Zinc-Pharo-Tests' 'Zinc-Pharo-Character-Encoding-Tests');
-            group: 'Character-Encoding' with: #('Zinc-Pharo-Character-Encoding-Core' 'Zinc-Pharo-Character-Encoding-Tests');
-            group: 'Resource-Meta' with: #('Zinc-Resource-Meta-Core' 'Zinc-Resource-Meta-Tests');
-            group: 'WebSocket' with: #('Zinc-Pharo-WebSocket-Core' 'Zinc-Pharo-WebSocket-Tests');
-            group: 'SSL' with: #('Zinc-Zodiac-Core' 'Zinc-Zodiac-Tests' 'Zinc-Pharo-Zodiac-Tests');
-            group: 'Zodiac' with: #('SSL') ].  "alias"
-
     spec for: #'gemstone' do: [
         spec baseline: 'GLASS1'
                  with: [ spec
-- 
2.34.1

