From 8f317ad74105f244309dff9e0fcb34a4d043c18c Mon Sep 17 00:00:00 2001
From: Patrik <development@finworks.biz>
Date: Mon, 17 Nov 2025 10:38:50 +0000
Subject: [PATCH 4/9] Fixing incorrect line ending

---
 .../BaselineOfZincHTTPComponents.class/instance/baseline..st    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/repository/BaselineOfZincHTTPComponents.package/BaselineOfZincHTTPComponents.class/instance/baseline..st b/repository/BaselineOfZincHTTPComponents.package/BaselineOfZincHTTPComponents.class/instance/baseline..st
index cd6c747c..e6b48be8 100644
--- a/repository/BaselineOfZincHTTPComponents.package/BaselineOfZincHTTPComponents.class/instance/baseline..st
+++ b/repository/BaselineOfZincHTTPComponents.package/BaselineOfZincHTTPComponents.class/instance/baseline..st
@@ -81,7 +81,7 @@ baseline: spec
             package: 'Zinc-GemStone-Resource-Meta-Core' with: [ spec requires: #('Zinc-Resource-Meta-Core' 'Zinc-GemStone-Character-Encoding-Core') ];
             package: 'Zinc-HTTP' with: [ spec requires: #('GLASS1' 'Zinc-FileSystem-Legacy');
                                               includes: #('SocketStream') ];
-            package: 'Zinc-Tests' with: [ spec requires: #('PharoCompatibility') ].
+            package: 'Zinc-Tests' with: [ spec requires: #('PharoCompatibility') ];
             package: 'Zinc-GemStone-HTTP' with: [ spec requires: #('Zinc-GemStone-Character-Encoding-Core' 'Zinc-GemStone-Resource-Meta-Core' 'Zinc-HTTP') ];
             package: 'Zinc-GemStone-HTTP-Examples' with: [ spec requires: #('Zinc-HTTP-Examples') ];
             package: 'Zinc-GemStone-Tests' with: [ spec requires: #('Zinc-Tests') ].
-- 
2.34.1

