From 145d459f56398643f3e1a8cf393cdaef0a22f759 Mon Sep 17 00:00:00 2001
From: Patrik <development@finworks.biz>
Date: Mon, 17 Nov 2025 13:53:38 +0000
Subject: [PATCH 9/9] Class must have a space between itself and message

---
 .../instance/writeResponseTerminationExceptionSet.st            | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/repository/Zinc-GemStone-HTTP.package/ZnMultiThreadedServer.extension/instance/writeResponseTerminationExceptionSet.st b/repository/Zinc-GemStone-HTTP.package/ZnMultiThreadedServer.extension/instance/writeResponseTerminationExceptionSet.st
index 3f13f915..7feaa915 100644
--- a/repository/Zinc-GemStone-HTTP.package/ZnMultiThreadedServer.extension/instance/writeResponseTerminationExceptionSet.st
+++ b/repository/Zinc-GemStone-HTTP.package/ZnMultiThreadedServer.extension/instance/writeResponseTerminationExceptionSet.st
@@ -4,5 +4,5 @@ writeResponseTerminationExceptionSet
   "Return the set of exceptions which, when they occur while writing a response,
   are interpreted as equivalent to a timeout or connection close."
 
-  ^ ZnUtilsexceptionSet:
+  ^ ZnUtils exceptionSet:
       #(#'ConnectionClosed' #'ConnectionTimedOut' #'PrimitiveFailed' #'SpSocketError')
-- 
2.34.1

